config/manager/config.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   namespace: numaplane-system
 data:
   config.yaml: |
-    clusterName: "CLUSTER_NAME_VALUE" # Replace with the cluster name where numaplane will get deployed.
```

Comment:
we are using a custom config for `make start`, so we don't need to patch anything for local